webapp/src/main/java/io/github/microcks/util/asyncapi/AsyncAPIImporter.java

```diff
@@ -437,22 +450,69 @@ private String getExamplePayload(JsonNode example) {
       return null;
    }
 
-   /** Check parameters presence into given operation node. */
-   private static boolean operationHasParameters(JsonNode operation) {
-      if (!operation.has("parameters")) {
-         return false;
-      }
-      Iterator<Entry<String, JsonNode>> parameters = operation.path("parameters").fields();
+   /** Check variables parts presence into given channel. */
+   private static boolean channelHasParts(String channel) {
+      return (channel.indexOf("/:") != -1 || channel.indexOf("/{") != -1);
+   }
```

Comment:
From all the example I've seen, I just encounter the `{}` notation for parameter. Is `:` a supported pattern?

Reply:
I copied this directly from some other place for REST/Sync APIs. You are right. this case doesn't apply AFAIK